### PR TITLE
ZRANGESTORE with special zset-max-ziplist-entries

### PR DESCRIPTION
**Pull request by AlonMagrafta**
@amiramm @adisht fyi

### CODE REVIEW

File: content/rs/release-notes/rs-6-2-10-february-2022.md [hunk 1]
```diff
@@ -130,6 +130,10 @@ For help upgrading a module, see [Add a module to a cluster](https://docs.redis.
 - RS72304 - Avoid starting a master shard when both master and replica shards crash and the replica did not finish recovery
 - RS74469 - Fix for some Redis Active-Active + Redis Streams scenarios that could lead to shard crash during backup; failure to backup
 
+## Known issues
+
+- The ZRANGESTORE command, with a special _zset-max-ziplist-entries_ configuration can crash Redis 6.2. See more on the [Redis repository 10767] (https://github.com/redis/redis/pull/10767)
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-2-10-february-2022.md:135]:
```suggestion
- The ZRANGESTORE command, with a special `_zset-max-ziplist-entries_` configuration can crash Redis 6.2. See more on the [Redis repository 10767] (https://github.com/redis/redis/pull/10767)
```

File: content/rs/release-notes/rs-6-2-4-august-2021.md [hunk 2]
```diff
@@ -157,7 +157,9 @@ To learn more, see [rladmin upgrade]({{<relref "/rs/references/rladmin.md#upgrad
 
 ## Known issues 
 
-A new command was added as part of Redis 6.2: [XAUTOCLAIM](https://redis.io/commands/xautoclaim/). When used in an Active-Active configuration, this command may cause Redis shards to crash, potentially resulting in data loss. The issue is fixed in Redis Enterprise Software version 6.2.12. Additionally, we recommend enabling AOF persistence for all Active-Active configurations.
+- A new command was added as part of Redis 6.2: [XAUTOCLAIM](https://redis.io/commands/xautoclaim/). When used in an Active-Active configuration, this command may cause Redis shards to crash, potentially resulting in data loss. The issue is fixed in Redis Enterprise Software version 6.2.12. Additionally, we recommend enabling AOF persistence for all Active-Active configurations.
+
+- The ZRANGESTORE command, with a special _zset-max-ziplist-entries_ configuration can crash Redis 6.2. See more on the [Redis repository 10767] (https://github.com/redis/redis/pull/10767)
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-2-4-august-2021.md:162]:
```suggestion
- The ZRANGESTORE command, with a `special _zset-max-ziplist-entries_` configuration can crash Redis 6.2. See more on the [Redis repository 10767] (https://github.com/redis/redis/pull/10767)
```

File: content/rs/release-notes/rs-6-2-8-october-2021.md [hunk 3]
```diff
@@ -96,7 +96,9 @@ All [known limitations]({{<relref "/rs/release-notes/rs-6-2-4-august-2021.md#kno
 
 ## Known issues 
 
-A new command was added as part of Redis 6.2: [XAUTOCLAIM](https://redis.io/commands/xautoclaim/). When used in an Active-Active configuration, this command may cause Redis shards to crash, potentially resulting in data loss. The issue is fixed in Redis Enterprise Software version 6.2.12. Additionally, we recommend enabling AOF persistence for all Active-Active configurations.
+- A new command was added as part of Redis 6.2: [XAUTOCLAIM](https://redis.io/commands/xautoclaim/). When used in an Active-Active configuration, this command may cause Redis shards to crash, potentially resulting in data loss. The issue is fixed in Redis Enterprise Software version 6.2.12. Additionally, we recommend enabling AOF persistence for all Active-Active configurations.
+
+- The ZRANGESTORE command, with a special _zset-max-ziplist-entries_ configuration can crash Redis 6.2. See more on the [Redis repository 10767] (https://github.com/redis/redis/pull/10767)
```
Comment (kaitlynmichael) [inline content/rs/release-notes/rs-6-2-8-october-2021.md:101]:
```suggestion
- The ZRANGESTORE command, with a special `_zset-max-ziplist-entries_` configuration can crash Redis 6.2. See more on the [Redis repository 10767] (https://github.com/redis/redis/pull/10767)
```

**Review by kaitlynmichael — APPROVED** [review]

**Review by rrelledge — APPROVED** [review]

File: content/rs/release-notes/rs-6-2-10-february-2022.md (hunk 1 above)
Comment (rrelledge) [reply content/rs/release-notes/rs-6-2-10-february-2022.md]: Fixed

File: content/rs/release-notes/rs-6-2-4-august-2021.md (hunk 2 above)
Comment (rrelledge) [reply content/rs/release-notes/rs-6-2-4-august-2021.md]: Fixed

File: content/rs/release-notes/rs-6-2-8-october-2021.md (hunk 3 above)
Comment (rrelledge) [reply content/rs/release-notes/rs-6-2-8-october-2021.md]: Fixed

**Pull request merged**